Restor/Controllers/ViewControllers/MainViewController.m

```diff
@@ -133,6 +132,20 @@ - (IBAction)image:(id)sender {
   if ([self.selectedImage.name isEqualToString:@"Custom Image"]) {
     [self showCustomImageView];
   } else {
+    // Check the config for changes before each image session, to avoid using outdated images.
+    NSError *error = [self.configController checkConfiguration];
```

Comment:
Could this be made to run at intervals instead of every single time? Imagine imaging 100 machines in a row, if every single time you start imaging it downloads and validates the cache that's at least 30s added to imaging, no?

Also, unless I'm mistaken, this misses the case where someone activates auto-image mode and leaves it running constantly as this method is only called when auto-image mode is started, not once for each disk.